### PR TITLE
feat: Expose addition test arguments on flame test

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

There were a few arguments in `test` that wasn't being exposed by `testWithGame` and `testWithFlameGame`.

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [ ] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

## Related Issues

Replace or remove this text.

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by luanpotter — APPROVED** [review]
LGTM

File: packages/flame_test/lib/src/test_flame_game.dart [hunk 1]
```diff
@@ -26,9 +26,23 @@ import 'package:meta/meta.dart';
 @isTest
 Future<void> testWithFlameGame(
   String testName,
-  AsyncGameFunction<FlameGame> testBody,
-) {
-  return testWithGame<FlameGame>(testName, FlameGame.new, testBody);
+  AsyncGameFunction<FlameGame> testBody, {
+  Timeout? timeout,
+  dynamic tags,
+  dynamic skip,
```
Comment (spydon) [inline packages/flame_test/lib/src/test_flame_game.dart:32]:
```suggestion
  Object? tags,
  Object? skip,
```

**Review by spydon — APPROVED** [review]

File: packages/flame_test/lib/src/test_flame_game.dart (hunk 1 above)
Comment (erickzanardo) [reply packages/flame_test/lib/src/test_flame_game.dart:32]: Shouldn't we match the api from test?

Comment (spydon) [reply packages/flame_test/lib/src/test_flame_game.dart:32]: They just have it as `dynamic` for legacy reasons, but we could, doesn't matter that much.

Comment (erickzanardo) [reply packages/flame_test/lib/src/test_flame_game.dart:32]: got ya, I slightly feel we would rather keep matching their apis. Gonna go with as is right now.

**Pull request merged**